packages/api-reference/src/components/Content/Schema/SchemaProperty.vue

```diff
@@ -107,6 +107,12 @@ const remainingEnumValues = computed(() =>
         #name>
         {{ name }}
       </template>
+      <template
+        v-if="value?.example"
+        #example>
+        Example:
+        {{ value.example }}
```

Comment:
We should probably use the mono space font for the example, I think that's what we do in the models.

https://github.com/scalar/scalar/blob/main/packages/api-reference/src/components/Content/Schema/SchemaProperty.vue#L123-L149

Also it's worth double checking that multiline examples don't break the layout (I think multiline examples are allowed?).

Reply:
I think instead we should use the following in compact too?
<img width="597" alt="image" src="https://github.com/user-attachments/assets/f2a6b6b8-d992-46f2-b90e-428aeb8292b4" />

Reply:
how do we feel instead about using the following for compact too:
<img width="597" alt="image" src="https://github.com/user-attachments/assets/2c6a9590-54b4-412d-8744-119df7a48ded" />

Reply:
cc @cameronrohani

Reply:
@cameronrohani is this good to go?

Reply:
i haven't the change to that yet, but can do - very quick thing to do

Reply:
@amritk i'll update accordingly and hyu!